vars/buildPlugin.groovy

```diff
@@ -282,6 +281,13 @@ def call(Map params = [:]) {
   }
 }
 
+private void discoverReferenceBuild() {
+  folders = env.JOB_NAME.split('/')
```

Comment:
Note that just `JOB_NAME` suffices

Reply:
Ok, I copied it from the `env.NODE_LABELS` part a couple of lines down.